proto/rill/runtime/v1/api.proto

```diff
@@ -397,6 +398,7 @@ message CreateInstanceRequest {
   map<string, string> variables = 7;
   map<string, string> annotations = 9;
   string frontend_url = 18;
+  string theme = 19;
```

Comment:
Can you remove the option here? Since the field gets populated from `rill.yaml`, it should not be possible to pass it over the APIs `CreateInstance` or `EditInstance` (because the YAML parser runs continuously and would quickly overwrite anything passed through the API)